backend/src/main/java/com/filegenie/backend/Services/JsonParserService.java

```diff
@@ -0,0 +1,40 @@
+package com.filegenie.backend.Services;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.filegenie.backend.Entities.Field;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class JsonParserService {
+
+    public List<Field> parseJsonToFields(File jsonFile) throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode rootNode = objectMapper.readTree(jsonFile);
+        return processNode(null, rootNode);
```

Comment:
Et là aussi pour que ça colle avec le controller:
```suggestion
    public List<Field> parseJsonToFields(File jsonFile) throws HttpException
        try {
            ObjectMapper objectMapper = new ObjectMapper();
            JsonNode rootNode = objectMapper.readTree(jsonFile);
            return processNode(null, rootNode);
        } catch (Exception e) {
            throw new HttpException(HttpStatus.INTERNAL_SERVER_ERROR, "Impossible de parser le fichier JSON: " + e.getMessage());
        }
```